File: src/main/java-graphql/io/github/graphql/java/model/DeletePullRequestReviewMutationResponse.java
package io.github.graphql.java.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult;
import java.util.Map;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-07T19:46:14+0800"
)
public class DeletePullRequestReviewMutationResponse extends GraphQLResult<Map<String, DeletePullRequestReviewPayloadTO>> {

    private static final String OPERATION_NAME = "deletePullRequestReview";

    public DeletePullRequestReviewMutationResponse() {
    }

    public DeletePullRequestReviewPayloadTO deletePullRequestReview() {
        Map<String, DeletePullRequestReviewPayloadTO> data = getData();
        return data != null ? data.get(OPERATION_NAME) : null;
    }

}

File: src/main/scala-graphql/io/github/graphql/model/DeletePullRequestReviewMutationResponse.scala
package io.github.graphql.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult
import java.util.{ Map => JMap }

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:01+0800"
)
class DeletePullRequestReviewMutationResponse extends GraphQLResult[JMap[String, DeletePullRequestReviewPayloadTO]] {

    def deletePullRequestReview(): DeletePullRequestReviewPayloadTO = {
        val data: JMap[String, DeletePullRequestReviewPayloadTO] = getData
        if (data != null) data.get(DeletePullRequestReviewMutationResponse.OPERATION_NAME) else null
    }

}

object DeletePullRequestReviewMutationResponse {

    private final val OPERATION_NAME: String = "deletePullRequestReview"

}